oauth2client/contrib/locked_file.py

```diff
@@ -347,10 +189,15 @@ def __init__(self, filename, mode, fallback_mode, use_native_locking=True):
         """
         opener = None
         if not opener and use_native_locking:
-            if _Win32Opener:
+            try:
+                from oauth2client.contrib._win32_opener import _Win32Opener
```

Comment:
Sorry for being vague. I propose this. Instead of defining

```python
class _FcntlOpener(_Opener):
    ...
```

in the `_fcntl_opener.py`, just define

```python
class _FcntlMixin(object):
    ...
```

and then in `locked_file.py` you could define

```python
class _FcntlOpener(_Opener, _FcntlMixin):
    ...
```

to get both feature sets in.